parcel.go

```diff
@@ -26,6 +38,13 @@ func (s ParcelStore) Get(number int) (Parcel, error) {
 	// заполните объект Parcel данными из таблицы
 	p := Parcel{}
 
+	row := s.db.QueryRow("SELECT number,client,address,status,created_at FROM parcel WHERE number = :number",
+		sql.Named("number", number))
+	err := row.Scan(&p.Number, &p.Client, &p.Address, &p.Status, &p.CreatedAt)
+	if err != nil {
+		return p, err
```

Comment:
Лучше возвращать пустые значения
```suggestion
		return Parcel{}, err
```